vars/pyenv.groovy

```diff
@@ -10,7 +10,9 @@ import com.ableton.VirtualEnv
  * @return New {@link com.ableton.VirtualEnv} instance.
  */
 VirtualEnv createVirtualEnv(String pythonVersion, String pyenvRoot = null) {
-  return new Pyenv(this, pyenvRoot ?: env.PYENV_ROOT).createVirtualEnv(pythonVersion)
+  return new Pyenv(
+    this, VirtualEnv.posixPath(pyenvRoot ?: env.PYENV_ROOT)
```

Comment:
Why do 2 conversions? One here and one in `Pyenv.createVirtualEnv`? Wouldn't it be simpler to do a single conversion in the constructor of the `Pyenv` object?

Reply:
I was trying to avoid doing too much work in the ctor. But I agree it is probably better in this case compared to the code duplication. Fixed in 735ff99.